<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TP_PlaylistWindow</class>
 <widget class="QWidget" name="TP_PlaylistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#777;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TP_TitleBar" name="frame_Title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #666;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_Title">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Playlist</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="TP_PlaylistContainer" name="playlistContainer" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="font">
         <font>
          <family>MT Extra</family>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="TP_PlaylistsWidget" name="playlistsWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_FileList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="TP_PlaylistBottomFrame" name="frame_Bottom">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>36</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #666;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="TP_PushButton" name="pushButton_Add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TP_PushButton" name="pushButton_Remove">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TP_PushButton" name="pushButton_Select">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TP_PushButton" name="pushButton_Sort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TP_PushButton" name="pushButton_Find">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="action_addFiles">
   <property name="text">
    <string>&amp;Files...</string>
   </property>
   <property name="toolTip">
    <string>Files</string>
   </property>
  </action>
  <action name="action_addFolder">
   <property name="text">
    <string>&amp;Folder...</string>
   </property>
  </action>
  <action name="action_addURL">
   <property name="text">
    <string>&amp;URL...</string>
   </property>
  </action>
  <action name="action_clearSelectedItems">
   <property name="text">
    <string>&amp;Selected items</string>
   </property>
  </action>
  <action name="action_clearUnselectedItems">
   <property name="text">
    <string>&amp;Unselected items</string>
   </property>
   <property name="toolTip">
    <string>Unselected items</string>
   </property>
  </action>
  <action name="action_clearInaccessibleItems">
   <property name="text">
    <string>&amp;Inaccessible items</string>
   </property>
  </action>
  <action name="action_clearAllItems">
   <property name="text">
    <string>&amp;All items</string>
   </property>
  </action>
  <action name="action_deleteFromDisk">
   <property name="text">
    <string>&amp;Delete from disk!</string>
   </property>
  </action>
  <action name="action_selectAll">
   <property name="text">
    <string>Select &amp;all</string>
   </property>
  </action>
  <action name="action_unselectAll">
   <property name="text">
    <string>&amp;Unselect all</string>
   </property>
  </action>
  <action name="action_reverseSelection">
   <property name="text">
    <string>&amp;Reverse selection</string>
   </property>
  </action>
  <action name="action_sortByDuration">
   <property name="text">
    <string>By &amp;duration</string>
   </property>
  </action>
  <action name="action_sortByPath">
   <property name="text">
    <string>By &amp;path (URI)</string>
   </property>
  </action>
  <action name="action_sortByFilename">
   <property name="text">
    <string>By &amp;filename</string>
   </property>
  </action>
  <action name="action_sortByDescription">
   <property name="text">
    <string>By &amp;description</string>
   </property>
  </action>
  <action name="action_setDescending">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Descending &amp;order</string>
   </property>
  </action>
  <action name="action_find">
   <property name="text">
    <string>&amp;Find...</string>
   </property>
  </action>
  <action name="action_findNext">
   <property name="text">
    <string>Find &amp;next</string>
   </property>
  </action>
  <action name="action_sortByTitle">
   <property name="text">
    <string>By &amp;title</string>
   </property>
  </action>
  <action name="action_sortByAlbum">
   <property name="text">
    <string>By albu&amp;m</string>
   </property>
  </action>
  <action name="action_sortByArtist">
   <property name="text">
    <string>By &amp;artist</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TP_TitleBar</class>
   <extends>QFrame</extends>
   <header>tp_titlebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TP_PushButton</class>
   <extends>QPushButton</extends>
   <header>tp_pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>TP_PlaylistContainer</class>
   <extends>QWidget</extends>
   <header>tp_playlistcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TP_PlaylistBottomFrame</class>
   <extends>QFrame</extends>
   <header>tp_playlistbottomframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TP_PlaylistsWidget</class>
   <extends>QListWidget</extends>
   <header>tp_playlistswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
